sale_order_import/readme/CONTRIBUTORS.rst

```diff
@@ -1,3 +1,5 @@
 * Alexis de Lattre <alexis.delattre@akretion.com>
 * Dennis Sluijk <d.sluijk@onestein.nl>
 * Andrea Stirpe <a.stirpe@onestein.nl>
+* Tam (Nguyen Duc) <tamnd@trobz.com>
```

Comment:
@phuctranerp sorry, bare migration does not entitle as contributors. To land in this list you should at least add something (eg: fix a bug, test coverage, new features, significant refactoring, etc).
Can you pls apply the same concept to all your PRs?

Reply:
Hi, thanks for your suggestion. I'll update it soon.

Reply:
updated